src/handwritten/_lib.py

```diff
@@ -148,39 +152,67 @@ def task_handle(self) -> type:
     @property
     def cal_handle(self) -> type:
         return CalHandle
-
+    
+    @property
+    def encoding(self):
+        if self._encoding is None:
+            self._import_lib()
+        return self._encoding
+        
     def _import_lib(self):
         """
         Determines the location of and loads the NI-DAQmx CAI DLL.
         """
         self._windll = None
         self._cdll = None
+        self._encoding = None
 
         windll = None
         cdll = None
-
-        if sys.platform.startswith('win') or sys.platform.startswith('cli'):
-            try:
-                if 'iron' in platform.python_implementation().lower():
+        encoding= None
+
+        # Feature Toggle to load nicaiu.dll or nicai_utf8.dll
+        # The Feature Toggle can be set in the .env file
+        load_dotenv()
```

Comment:
FYI, in MeasurementLink Python I had to do extra work to find the `.env` file in a reasonable location for TestStand code modules and PyInstaller EXEs: https://github.com/ni/measurementlink-python/blob/main/ni_measurementlink_service/_dotenvpath.py

As long as this nidaqmx-python feature toggle is only used for testing, I don't think we need it to be compatible with these deployment scenarios.